index.html

```diff
@@ -775,5 +775,104 @@ <h3 id="placement-of-combination-of-group-ruby-and-group-ruby">Placement of the
 described in [[[#placement-of-group-ruby]]].</p>
 </section>
 </section>
+
+
+
+<section class="appendix">
+<h2 id="simplification-of-ruby-placement-rules">Simplification of Ruby placement rules</h2>
+
+<section>
+<h3 id="alignment-between-ruby-annotation-and-base-text">Alignment between ruby annotation and base text</h3>
+
+<section>
+<h4 id="alignment-between-ruby-annotation-and-base-text-mono-ruby">Mono-ruby</h4>
+<p>In <a title="mono-ruby">mono-ruby</a>, 
+following three methods are mainly used for the relative positions of the ruby annotation and its base text.
+In this document, option 1 is picked. Characters of the ruby annotation are set solid.
+</p>
+<ol>
+  <li id="l20211116001">The center of the ruby annotation and of its base text are aligned in the inline direction.</li>
+  <li id="l20211116002">The beginning of the ruby annotation and of its base text are aligned in the inline direction.</li>
+  <li id="l20211116003">In principle, the beginning of the ruby annotation and of its base text are aligned in the inline direction.
+    If the ruby annotation is wider than its base text, 
+    follow <a href="#l20211116033">option 3</a> in [[[#overhang-surrounding-characters]]] with following direction as prioritized.
+    The results may depend on the character class of characters preceding and following.</li>
+</ol>
+<p>In some cases, option 2 may be picked for every <a title="ruby block">ruby blocks</a> at line edge.
+Alignment between ruby annotation and base text may differ in this case.</p>
+</section>
+
+<section>
+<h4 id="alignment-between-ruby-annotation-and-base-text-group-ruby">Group-ruby</h4>
+<p>In Grouo-ruby, when ruby annotation and base text are different length in solid setting,
```

Comment:
```suggestion
<p>In Group-ruby, when ruby annotation and base text are different length in solid setting,
```

Reply:
"length" should be "lengths"?

Reply:
wondered about that.. Actually, "when lengths of ruby annotation and base text are different" is better??

Reply:
or `when lengths of ruby annotation and base text differ`??